docs/index.md

```diff
@@ -1128,6 +1128,50 @@ app/components
 </div>
 ```
 
+##### Translations (experimental)
+
+To use experimental support for `I18n` translations, include `ViewComponent::SidecarI18n`:
```

Comment:
What do you think about renaming this to `Translatable` be more idiomatic Ruby?

```suggestion
To use experimental support for `I18n` translations, include `ViewComponent::Translatable`:
```

Reply:
That's a great suggestion, it fits well with existing modules 👍